elements/lisk-chain/src/account.ts

```diff
@@ -16,9 +16,6 @@ import { AccountJSON } from './types';
 export const accountDefaultValues = {
 	publicKey: undefined,
 	// tslint:disable-next-line:no-null-keyword
-	secondPublicKey: null,
-	secondSignature: 0,
```

Comment:
The same applies here, this issues should remove second-signature from everwhere we are generating transactions. But until we completely remove it from database we should not remove it from these files.